moveit_core/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ find_package(trajectory_msgs REQUIRED)
 find_package(visualization_msgs REQUIRED)
 find_package(common_interfaces REQUIRED)
 find_package(pluginlib REQUIRED)
+find_package(generate_parameter_library REQUIRED)
```

Comment:
Can you try to copy PR #2091 as closely as possible, line-for-line. For example, this line isn't alphabetized.

Reply:
Fixed